app/views/application/_footer.erb

```diff
@@ -10,6 +10,9 @@
         <p>
           <a href="/about">About</a>
         </p>
+        <p>
+          <a href="/contribute">Contribute to Exercism</a>
```

Comment:
I like it. It's really different from "how to use the site" which is what that other group of links is about.